config/settings.yml

```diff
@@ -1,3 +1,7 @@
+sdr_api:
+  sweep_uploads_older_than: '7 days ago'
```

Comment:
Can we make this an integer and get rid of the dependency on chronic?

Reply:
@jcoyne I pondered that, but... what would the integer mean? # of days? What if we want to switch to two weeks, for example? On one hand, I like being able to push changes like this by changing configuration alone, which wouldn't be possible if we switch this to an integer. On the other hand, changing days to weeks (or months or whatever) an easy code change.

Reply:
@jcoyne FWIW, the `whenever` gem already adds `chronic` as a dependency. Still worth making this change?

Reply:
done.